deployment/terraform/firewall.tf

```diff
@@ -404,4 +404,24 @@ resource "aws_security_group_rule" "batch_msk_egress" {
 
   security_group_id        = aws_security_group.batch.id
   source_security_group_id = aws_security_group.msk.id
-}
\ No newline at end of file
+}
+
+resource "aws_security_group_rule" "opensearch_egress" {
+  type      = "egress"
+  from_port = 9200
+  to_port   = 9200
+  protocol  = "tcp"
+
+  security_group_id        = module.database_enc.database_security_group_id
+  source_security_group_id = aws_security_group.opensearch.id
+}
+
+resource "aws_security_group_rule" "opensearch_ingress" {
```

Comment:
Could you explain the reason behind creating an ingress rule for the database to enable communication with OpenSearch? OpenSearch doesn't communicate with the DB.

Reply:
Refactored in favor of logstash `source_security_group_id`